.github/workflows/verify.yml

```diff
@@ -72,3 +74,19 @@ jobs:
           java-version: '11'
           cache: 'sbt'
       - run: sbt checkScalaFormat
+
+  website-prettier:
```

Comment:
Not a blocker, but we could maybe add a check here to only run when there are any `*.yml`, `*.md`, or `*.js` files changed.

Reply:
I'm not sure what you mean. If there are files of a type that prettier doesn't support, prettier doesn't run on them. And by definition no other file types will be places there, because why would they? Do you mean running this entire check conditionally? (I'm not sure if that's possible) Or do you mean running prettier only on files changed in that specific commit being checked?

Reply:
yea I meant the entire thing, but no worries at all. Just ignore this.